pygluu-containerlib/pygluu/containerlib/persistence/sql.py

```
@@ -201,61 +205,61 @@ def insert_into(self, table_name, column_mapping):
 
         query = table.insert().values(column_mapping)
         with self.engine.connect() as conn:
-            try:
-                conn.execute(query)
-            except Exception as exc:  # noqa: B902
-                self.adapter.on_insert_into_error(exc)
+            with conn.begin():
+                try:
+                    conn.execute(query)
+                except Exception as exc:  # noqa: B902
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)